agent/src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -122,19 +123,20 @@ public static void fuzzerInitialize(String[] args) {
       }
       System.exit(1);
     }
-    List<Class<?>> alwaysIgnore =
-        Arrays.stream(args)
-            .skip(1)
-            .map(name -> {
-              try {
-                return ClassLoader.getSystemClassLoader().loadClass(name);
-              } catch (ClassNotFoundException e) {
-                System.err.printf("Failed to find class '%s' specified in --autofuzz_ignore", name);
-                System.exit(1);
-              }
-              throw new Error("Not reached");
-            })
-            .collect(Collectors.toList());
+
+    Set<Class<?>> alwaysIgnore = new HashSet<>();
+    ClassGraph classGraph = new ClassGraph().enableClassInfo().enableSystemJarsAndModules();
+    try (ScanResult result = classGraph.scan()) {
+      List<Class<?>> allExceptions = result.getSubclasses(Throwable.class).loadClasses();
+      List<Pattern> patterns =
+          Arrays.stream(args).skip(1).map(Pattern::compile).collect(Collectors.toList());
```

Comment:
Note that most Java class names contain '.', which has a special meaning here. Not sure how to handle this in the best way, but maybe we could just extract and reuse `com.code_intelligence.jazzer.instrumentor.SimpleGlobMatcher` here instead of using full-blown regexes?

Reply:
Changed the logic to use `SimpleGlobMatcher`